src/pybind/mgr/cephadm/module.py

```diff
@@ -2645,6 +2650,7 @@ def _apply(self, spec: ServiceSpec) -> str:
                 'mgr': PlacementSpec(count=2),
                 'mds': PlacementSpec(count=2),
                 'rgw': PlacementSpec(count=2),
+                'iscsi': PlacementSpec(count=1),
```

Comment:
Not sure what the default would be. I assume for HA maybe this should be 2. But keeping it simple first.

Reply:
similar issue for `nfs` which also uses a default value of 1